File: LocaNet/src/CadastroImoveis.java
import java.util.ArrayList;
import java.util.List;

public class CadastroImoveis {
    
    private List<Imovel> cadastro;

    public CadastroImoveis() {
        this.cadastro = new ArrayList<>();
    }

    public List<Imovel> getCadastro() {
        return this.cadastro;
    }

    public void setCadastro(List<Imovel> cadastro) {
        this.cadastro = cadastro;
    }

    public void adicionarImovel(Imovel imovel) {
        this.cadastro.add(imovel);
    }

}

File: LocaNet-PyVersion/CadastroImoveis.py
#Classe CadastroImóveis
class CadastroImoveis:

  def __init__(self):
    self.cadastro = []

  def adicionarImovel(self, imovel):
    self.cadastro.append(imovel)

  def buscarImovel(self, numIPTU):
    for imovel in self.cadastro:
      if imovel.getNumIPTU() == numIPTU:
        return imovel
      return None